dclab/definitions/meta_const.py

```diff
@@ -124,6 +125,25 @@
         ["target duration", float, "Target measurement duration [min]"],
         ["target event count", fint, "Target event count for online gating"],
     ],
+    # All qpi-related keywords
+    "qpi": [
+        # experiment-related qpi metadata
+        ["wavelength", float, "Laser wavelength [nm]"],
+        ["medium index", float, "Refractive index of Medium"],
+        # post-analysis-related qpi metadata
+        ["software version", str, "Software version used"],
+        # calculation of pha and amp from hologram
+        ["filter_name", str, "Fourier filter used"],
+        ["filter_size", str, "Fourier filter size [1/pix]"],
+        ["filter_size_interpretation", str, "How to interpret filter size"],
+        ["sideband_freq", f2dfloatarray,
+         "Frequency coordinates of the sideband [1/pix]"],
+        # processing of hologram
+        ["padding", fbool, "If padding was used"],
```

Comment:
Padding is an integer.

Reply:
I know this is a different package, but I use it. PAdding here is a bool: https://qpimage.readthedocs.io/en/stable/_modules/qpimage/core.html?highlight=padding#

Reply:
Historically, `padding` was inintially boolean, but then it turned out that increasing the padding size yields cleaner results at the cost of more computational power required. Hence the confusion in the docstrings. Yes, the docs should be update in qpretrieve/qpimage.